modules/base/mw.cx.SiteMapper.js

```
@@ -143,6 +156,10 @@ mw.cx.SiteMapper = class {
 		if ( mw.cx.getCXVersion() === 2 ) {
 			cxserverURL = cxserverURL.replace( 'v1', 'v2' );
 		}
+		// if module has /mdwiki then replace it with /en
+		if (module.indexOf('/mdwiki') > -1) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)